src/Resources/config/core.php

```diff
@@ -124,6 +125,35 @@
 
         ->alias(FormLabelTranslatorStrategy::class, 'sonata.admin.label.strategy.form_component')
 
+        ->set('sonata.admin.controller.crud', CRUDController::class)
+            ->public()
+            ->args([
+                new ReferenceConfigurator('request_stack'),
```

Comment:
Where can I learn more about `ReferenceConfigurator`? Why should we use it rather than just `Reference`?

Reply:
I don't know why. I use it to be consistent with what @franmomu used before.

Reply:
See https://github.com/sonata-project/SonataAdminBundle/pull/6284#discussion_r470606932, `ref` and `service` instantiate `ReferenceConfigurator`.

Reply:
I'm asking because it does not seem to be documented at all :sweat_smile:

Reply:
Please see https://symfony-devs.slack.com/archives/C8WHX21K7/p1599807202321700

Reply:
TL;DR you are correct, this is the way :)